frontend/src/v4/routes/components/criteriaField/newCriterionForm/newCriterionForm.component.tsx

```diff
@@ -31,6 +34,8 @@ export const NewCriterionForm = ({ criterion, onSubmit, alreadySelectedFilters =
 	// used to clear the form after saving
 	const [key, setKey] = useState(uniqueId());
 
+	const theme = _.merge(_.cloneDeep(v4Theme), { palette });
```

Comment:
I think you might only want to do this if it's V4? Because currently it's giving V5 some V4 styling
![image](https://github.com/3drepo/3drepo.io/assets/62481250/3277f805-9cfc-4b97-b234-0fd12c6384d3)